index.html

```diff
@@ -3853,16 +3853,16 @@ <h3>Reserved Extension Points</h3>
 
         <p>
 This specification reserves a number of [=properties=] to serve as possible
-extension points. While some implementers signaled interest in these properties,
-their inclusion in this specification was considered to be premature; these
-extension points might be more formally defined in future versions of this
-specification. It is important to note that these properties are not defined by
-this specification and implementers are cautioned that usage of these properties
-is considered experimental.
+extension points. While some implementers signalled interest in these properties,
+their inclusion in this specification was considered to be premature. Some of these
+extension points were originally defined in previous versions of this
+specification, whilst others were not. It is important to note that none of these properties
+are defined by this specification. Consequently, implementers are cautioned that usage of these 
+properties is considered experimental.
```

Comment:
W3C uses [US English](https://www.w3.org/Guide/manual-of-style/#Spelling).
```suggestion
extension points. While some implementers signaled interest in these properties,
their inclusion in this specification was considered to be premature. Some of
these extension points were originally defined in previous versions of this
specification, while others were not. It is important to note that none of these
properties are defined by this specification. Consequently, implementers are
cautioned that use of these properties is considered experimental.
```

Reply:
Signaled and signalled are both English terms. Signaled is predominantly used in 🇺🇸 American (US) English ( en-US ) while signalled is predominantly used in 🇬🇧 British English (used in UK/AU/NZ) ( en-GB ). (see https://sapling.ai/usage/signaled-vs-signalled)

While and whilst are the same, but whilst is a bit more formal (according to one famous grammar book). (see https://www.ielts-simon.com/ielts-help-and-english-pr/2010/11/ielts-grammar-while-whilst-whereas-as.html)

I gladly accept replacing usage by use. Note that the original text contained usage and was not edited by this PR.

Note that the rest of the proposed changes are line feeds in different places.

Reply:
https://www.grammarly.com/blog/while-vs-whilst/
> Typically, Brits use whilst and Americans use while.

https://thecontentauthority.com/blog/signalled-vs-signaled
> The word “signalled” is the British spelling of the word “signaled”.

Again, W3C uses **US English**.  In other words, the changes to "signaled" and "while" are appropriate.